proxy/indexer/indexer_base.py

```diff
@@ -63,7 +63,7 @@ def run(self):
             try:
                 self.process_functions()
             except BaseException as exc:
-                LOG.debug('Exception on transactions processing.', exc_info=exc)
+                LOG.warn('Exception on transactions processing.', exc_info=exc)
```

Comment:
`warn` method is deprecated, you should use `warning`

Reply:
Fixed